spec/inputs/s3_spec.rb

```diff
@@ -245,7 +248,7 @@
       let(:log_file) { File.join(File.dirname(__FILE__), '..', 'fixtures', 'cloudfront.log') }
 
       it 'should extract metadata from cloudfront log' do
-        events = fetch_events(settings)
+        events = fetch_events(config)
 
         events.each do |event|
           expect(event['cloudfront_fields']).to eq('date time x-edge-location c-ip x-event sc-bytes x-cf-status x-cf-client-id cs-uri-stem cs-uri-query c-referrer x-page-url​  c-user-agent x-sname x-sname-query x-file-ext x-sid')
```

Comment:
@purbon 
renaming settings to config this is something I was concerned about doing.
I decided to change it because the `devutils` helper is using `config` to initialize his pipeline and I wanted consistency :(